src/test/java/ir/sahab/dockercomposer/DockerComposeRunnerTest.java

```diff
@@ -15,6 +16,39 @@
     private static final int PORT = 2181;
     private static final Condition<Service> accessible = new ServiceAccessible(PORT);
 
+    @Test
+    public void testServiceWithInjectedUserId() throws Exception {
+        DockerComposeRunner runner = null;
+        try {
+            // Start the zookeeper service with injected uid
+
+            Path file = Paths.get(getClass().getResource("/docker-compose/j-group-with-uid.yaml").getFile());
+            runner = new DockerComposeRunner(file);
+            Map<String, Service> services = runner.start(true);
+            Service jGroupService = services.get("jgroup-with-uid");
+
+            WaitFor.portOpen("jgroup-with-uid", PORT).process(services);
+            assertThat(jGroupService).isNotNull();
+            String containerId = jGroupService.getId();
+            //Get user id in machine
+
+            ProcessExecutor executor = new ProcessExecutor("id", "-u");
+            String expectedUserId = executor.readOutput(true).execute().outputString().trim();
+            //Get user id inside the container
+
+            executor = new ProcessExecutor("docker", "exec", containerId, "id", "-u");
+            String containerUserId = executor.readOutput(true).execute().outputString().trim();
+            Assert.assertEquals(expectedUserId, containerUserId);
+        }
+        catch (Exception ex){
+            throw ex;
+        }
+        finally {
```

Comment:
Write finally in the same line of '{' (code style).

Reply:
Done